### PR TITLE
Feature/flickr api

### PR DESCRIPTION
**Pull request by rschumann**

### CODE REVIEW

File: backend/app/helpers/backend/forms_helper.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+module Backend
+  module FormsHelper
+    def is_cover_photo?(media_content, photo)
+      media_content.try(:main_photo_id) == photo.try(:photo_id) if media_content.try(:main_photo_id).present?
```
Comment (simaob) [inline backend/app/helpers/backend/forms_helper.rb]: suggest replacing with:

`media_content.main_photo_id.present? && photo.photo_id.present? && media_content.main_photo_id == photo.photo_id`

No need to use try if you are passing an object and checking an attribute of that object, I think.

File: backend/app/models/album.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+class Album < ApplicationRecord
+  include Sanitizable
+
+  belongs_to :media_content, foreign_key: :media_content_id, touch: true
+
+  validates :photoset_id,  presence: true
+  validates :photoset_url, presence: true
+
+  scope :is_cover_in_albums, -> (main_photo_id) { select(:main_photo_id).where(main_photo_id: main_photo_id) }
```
Comment (simaob) [inline backend/app/models/album.rb]: I think that this shouldn't be an Album scope but rather a method in the MediaContent model?

```
def is_cover_in_albums?
  relation.any?  #whatever the relation is that links MediaContent to Album as cover.
end
```

File: backend/app/models/album_relation.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#
+# Indexes
+#
+#  index_album_relations_on_album_photo_id                  (album_photo_id)
+#  index_album_relations_on_photoset_id                     (photoset_id)
+#  index_album_relations_on_photoset_id_and_album_photo_id  (photoset_id,album_photo_id) UNIQUE
+#
+
+class AlbumRelation < ApplicationRecord
+  belongs_to :photoset,    class_name: 'MediaContent', foreign_key: :photoset_id,    touch: true
```
Comment (simaob) [inline backend/app/models/album_relation.rb]: no need to specify the foreign_key, as it's inferred from the relation.

File: backend/app/views/backend/activities/index.html.slim [hunk 4]
```diff
@@ -0,0 +1,16 @@
+h2 Activities
```
Comment (simaob) [inline backend/app/views/backend/activities/index.html.slim]: I mentioned in a previews PR review that I would like to not use slim as the template engine.

**Comment by dhakelila** [comment]
Should we merge this???

**Pull request merged**